config/routes/spree.rb

```diff
@@ -79,6 +92,13 @@
         end
       end
     end
+
+    resources :users do
+      member do
+        put :generate_api_key
+        put :clear_api_key
+      end
+    end
```

Comment:
and why these new routes if we are only moving API controllers :thinking: ?

Reply:
these non-api routes are related to api and are defined in spree_api [here](https://github.com/openfoodfoundation/spree/blob/fe0a1311abb097bf3ac8001a24246c6cff5ecdbb/api/config/routes.rb#L3)
and the code also comes from spree_api [here](https://github.com/openfoodfoundation/spree/blob/master/api/app/models/spree/user_decorator.rb)